tests/clear-symver.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/sh -e
+SCRATCH=scratch/$(basename $0 .sh)
+
+rm -rf ${SCRATCH}
+mkdir -p ${SCRATCH}
+
+cp main ${SCRATCH}/
+RANDOM_PATH=$(pwd)/${SCRATCH}/$RANDOM
+
+SYMBOL_TO_REMOVE=__libc_start_main
+VERSION_TO_REMOVE=GLIBC_2.34
+
+readelfData=$(readelf -V ${SCRATCH}/main 2>&1)
+
+if [ $(echo "$readelfData" | grep --count "$VERSION_TO_REMOVE") < 2 ]; then
+    # We expect this to appear at least twice: once for the symbol entry,
+    # and once for verneed entry.
+    echo "Couldn't find expected versioned symbol. Are you building with glibc?"
+    exit 1
```

Comment:
Sorry for the late review, but never now than never...
I think this test should not fail on non-glibc machines as we also have openbsd/freebsd user:

```suggestion
    exit 0
```

Reply:
Good idea. I've changed this, and improved the message printed.

Reply:
Can you still make the test not fail in this case?

Reply:
Whoops — I didn't notice that git was stuck asking for the password to push this. I've actually uploaded the new version with that changed. Sorry!